wdk-ddi-src/content/wdm/nc-wdm-pob_pre_operation_callback.md

```diff
@@ -47,6 +47,7 @@ api_name:
 ## -description
 
 The <i>ObjectPreCallback</i> routine is called by the operating system when a process or thread handle operation occurs.
+<div class="alert"><b>Warning</b> The actions that you can perform in this callback are restricted for safe calls. See <a href="/windows-hardware/drivers/kernel/windows-kernel-mode-process-and-thread-manager#best">Best Practices</a></div>

```

Comment:
Not seeing a subsection with the anchor Best in the target file.  Is that work in progress?